test/torch/tensors/test_logging.py

```diff
@@ -117,3 +121,16 @@ def test_remote_function_on_log_chain(workers):
     y = y_ptr.get()
 
     assert (y.child.child == x_tensor.add(x_tensor)).all()
+
+
+def test_print_log_chain():
+    """
+    Test sending and getting back a chain including a logtensor
+    """
+
+    # Build a long chain tensor Wrapper>LoggingTensor>TorchTensor
+    x_tensor = torch.Tensor([1, 2, 3])
+    x = LoggingTensor().on(x_tensor)
+
+    assert isinstance(x.__str__(), str)
```

Comment:
we should test that we are actually printing the right string

Reply:
Printing is hard because it changes a lot (we just changed pointer printing), meaning that we need to rewriting the tests very often and copy pasting strings (with ids in it etc) is painful.

Reply:
How often do we plan on changing this? It seems to me like something that would not change often

Reply:
If history is any indicator - once every couple months

Reply:
lol